log/log.go

```diff
@@ -41,31 +40,50 @@ var levelPrefix = [...]string{
 var (
 	// Level stores the current logging level.
 	Level = LevelInfo
-	// SysLogger is a syslog Writer to be used if not nil.
-	SysLogger *syslog.Writer
+	// output stores a SetLogger() parameter.
+	output SyslogWriter
```

Comment:
So... `output` probably wasn't the best suggestions (There's already a function named that)
Let's try: `syslogWriter`?

Also nitpick: I would move the type declaration before the variable declaration.